tests/units/common/test_cleanup.py

```diff
@@ -97,9 +97,9 @@ def test_convert_html_subsripts_to_latex(test_input, expected):
 
 
 xml = "<div><p>example<h1> h1 example</h1></p></div>"
-xml_just_p = "<p>example h1 example</p>"
-xml_just_div = "<div>example h1 example</div>"
-xml_div_and_h1 = "<div>example<h1> h1 example</h1></div>"
+xml_just_p = "<p>example\n h1 example</p>"
+xml_just_div = "<div>\nexample\n h1 example</div>"
+xml_div_and_h1 = "<div>\nexample<h1> h1 example</h1></div>"
```

Comment:
hmm are these changes related to poetry?

Reply:
kind of yes. The upgrading airflow also upgrades the dependencies. As a result, the version of `bleach` upgraded as well, and it works slightly different than the older one

Reply:
Thanks for the explanation, but still I don't understand why the new lines. Do we have to configure it differently?

Reply:
No, there was no changes made from our side for bleach

Reply:
Yes I understand that we didn't do any changes and it comes from bleach. What I'm saying is, if we should configure bleach differently, since it seems wrong to have new lines.

Reply:
I don't like it as well. The new line appears instead of the cleaned starting tag, when we are cleaning the tags which are not surrounded. Let's say we have:
`<div><p></p></div>`
we clean the string from "p", the result will be:
`<div>\n</div>`

Reply:
We have to understand the impact of this change, we have quite a few functions in cleanup that uses bleach.

Reply:
Should we try to downgrade the bleach version? Because I spent a day in order to understand what actually changed, what we are getting the newlines introduced. However, I wasn't able to find something about this in the documentation

Reply:
I cannot find anything on the change log regarding the new lines.
They already appear on 5.0.0 version 
https://bleach.readthedocs.io/en/latest/changes.html#version-5-0-0-april-7th-2022

The PR which as related to clean function, should not make a difference. https://github.com/mozilla/bleach/pull/644/files

Reply:
So, what would be a better option:
1. Downgrade to 4.0.1 (now bleach version is 6.0.0)
2. Keep the newest versions as a cost with new lines (as it is now) in our tests